Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    const string SPEED = "Speed";
+    const string CROUCH = "Crouch";
+    const string JUMP = "Jump";
+
+    [SerializeField] private float speed;
+    [SerializeField] private float jumpPower;
+    
+    Animator animator;
+    Rigidbody2D rb;
+    bool isGrounded;
+    private void Start() {
+        animator = GetComponent<Animator>();
+        rb = GetComponent<Rigidbody2D>();
+        isGrounded = true;
+    }
+    private void Update() {
+        float horizontal = Input.GetAxisRaw("Horizontal");
+        float vertical = Input.GetAxisRaw("Vertical");
+        Crouch();
+        Jump();
+        MoveHorizontal(horizontal);
+        HorizontalAnim(horizontal);
```

Comment:
Call HorizontalAnim inside the MoveHorizontal method since the parameter horizontal is same for both method which will increase code readability.